asset-transfer-basic/chaincode-typescript/package.json

```diff
@@ -12,18 +12,22 @@
         "lint": "tslint -c tslint.json 'src/**/*.ts'",
         "pretest": "npm run lint",
         "test": "nyc mocha -r ts-node/register src/**/*.spec.ts",
-        "start": "fabric-chaincode-node start",
+        "start": "set -x fabric-chaincode-node start",
         "build": "tsc",
         "build:watch": "tsc -w",
-        "prepublishOnly": "npm run build"
+        "prepublishOnly": "npm run build",
```

Comment:
Eventually I think we'll want a sample in each chaincode language.
Therefore, should we add new ccaas samples for each language rather than simply changing chaincode-typescript? Is there a way to have a single version of the chaincode logic (invokes) but have a "traditional" wrapper and a ccaas wrapper, so that it can be started either way?

Reply:
To clarify, what if people attempt to run chaincode-typescript the old way? Don't we need support for both the old way and the new way depending on the flag passed?

Reply:
It'll work :-) They'd use the `deployCC` command, but it'll work the same.

For nodejs so long as the `start` script is still there `"start": "set -x fabric-chaincode-node start",` as it is in the example. 

Similarly (or maybe even easier) with Java.  Just use it as is.  All the smarts are internalised